selftests/functional/test_lv_utils.py

```diff
@@ -114,6 +114,8 @@ class DiskSpace(unittest.TestCase):
                      'macOS does not support scsi_debug module')
     @unittest.skipIf(not process.can_sudo(), "This test requires root or "
                      "passwordless sudo configured.")
+    @unittest.skipIf(process.system("which modprobe", ignore_status=True),
+                     "kmod not installed (command modprobe is missing)")
```

Comment:
This is fine. However we have now, 4 definitions of `def missing_binary()` and now this new method. So maybe would be the time to merge them into `selftests/utils.py`. Just a suggestion, not a blocker.

Reply:
I'll change it. and it's not new, it's also at https://github.com/avocado-framework/avocado/blob/master/selftests/functional/test_lv_utils.py#L25
We'll need a consolidation of all this once we move the tests to avocado.utils